CollAction/Views/Shared/_Layout.cshtml

```diff
@@ -58,7 +58,7 @@
                             <li><a asp-area="" asp-controller="Projects" asp-action="Find">Find Project</a></li>
                             <li><a asp-area="" asp-controller="Projects" asp-action="StartInfo">Start Project</a></li>
                             <li><a asp-area="" asp-controller="Home" asp-action="About">About</a></li>
-                            @if (User.IsInRole(Constants.AdminRole))
+                            @if (User.IsInRole("Admin"))
```

Comment:
Why? The constants ensure the admin role is "typo-checked".

Reply:
A that slipped in.. it's wouldn't compile on my machine. No idea why; probably a version mixup. I'll put some more time in that.

Reply:
Compiles now... no idea why.